src/xrEngine/xr_ioc_cmd.cpp

```diff
@@ -881,6 +881,7 @@ void CCC_Register()
     // Texture manager
     CMD4(CCC_Integer, "texture_lod", &psTextureLOD, 0, 4);
     CMD4(CCC_Integer, "net_dedicated_sleep", &psNET_DedicatedSleep, 0, 64);
+    CMD3(CCC_Mask, "texture_disable_managed", &psDeviceFlags, rsDisableManagedTex);
```

Comment:
There's a reason to keep them separated so that console command may be enabled by the user, but disabled by renderer if some additional requirements are not met.
How it works:
Renderer checks for user options (flags) and enables `o` options with some additional checks.

Reply:
Also, it's better to rename it to something like `r__no_texture_staging` and move it to xrRender_Console.cpp

Reply:
Sounds right, agree